user-guide/query-examples-hacks/sunbusrt-sankey-example.md

```diff
@@ -0,0 +1,105 @@
+# Funnel Queries (Sankey and Sunbusrt Visualizations)
+
+Sankey and Sunburst visualizations are very good for understanding behavioral flows and sequences.
+
+To create such a chain of events you'll need to define the actions/event you want to examine, define how many events are in a sequence (max number of events) and assign chronological session numbers per user.
+
+Here is a step by step example using the `WITH` clause in PostgreSQL, you can view live renders of the visualizations [in our demo account](http://demo.redash.io/queries/2280#3113).
+
+  * We used the `WITH` clause in this example - `WITH` clauses allow you to name a sub-query block, this way your query is modular (and also runs faster), it can later be referred to inside the main query instead of making a large main query with many aliases, `JOIN`s  or other complexities. You can have multiple sub-queries, just be sure to have a coma between them.
```

Comment:
what about (and sometimes runs faster)? We did notice that happening, can be a good tip or something to have in mind if the user experiences slow running times for giant queries